.travis.yml

```diff
@@ -3,7 +3,9 @@ sudo: false
 cache: bundler
 
 rvm:
-  - 2.3.0
+  - 2.3.3
+  - 2.4.1
 
 env:
+  - RAILS_VERSION=4.2
   - RAILS_VERSION=5.0
```

Comment:
:heart::heart::heart::heart::clap::clap::clap::clap:

There are many warnings, but this verifies merit against the latest Rails and Ruby versions. So happy. I thought the `belongs_to` association didn't work but now it seems to work well.

:heart::heart::heart::heart::clap::clap::clap::clap: